lib/JsonNode.h

```diff
@@ -118,11 +118,12 @@ class DLL_LINKAGE JsonNode
 		h & type;
 		switch (type) {
 			break; case DATA_NULL:
-			break; case DATA_BOOL:   h & data.Bool;
-			break; case DATA_FLOAT:  h & data.Float;
-			break; case DATA_STRING: h & data.String;
-			break; case DATA_VECTOR: h & data.Vector;
-			break; case DATA_STRUCT: h & data.Struct;
+			break; case DATA_BOOL:    h & data.Bool;
+			break; case DATA_INTEGER: h & data.Integer;
+			break; case DATA_FLOAT:   h & data.Float;
+			break; case DATA_STRING:  h & data.String;
+			break; case DATA_VECTOR:  h & data.Vector;
+			break; case DATA_STRUCT:  h & data.Struct;

```

Comment:
If this all to be reformatted then please check:
https://wiki.vcmi.eu/Coding_guidelines#Switch_statement

Reply:
Ah, sorry, now I see that I need to revert changes in this file since there IS a check on DATA_INTEGER a bit below. Anyway I'm interested how these "forwarded" `break`s fit in your styleguide? I mean, does styleguide allow them? They a bit "shift" `case` word to the right) And how about oneliners? Is there need to place things like `h & data.Bool;` on a newline or it's fine as it is?

Reply:
Just revert changes

Reply:
Sure. Just curious

Reply:
No this do not fit to styleguide.